test/config/MockAppConfig.scala

```diff
@@ -86,5 +88,7 @@ class MockAppConfig extends AppConfig {
 
     override val useGetCalcListIFPlatform: Boolean = true
 
+    override def emaSupportingAgentsEnabled: Boolean = false
```

Comment:
defined 2 times, please remove one of them

Reply:
Both `emaSupportingAgentsEnabled` definitions are needed for this file as removing either one will cause an error